app/__init__.py

```diff
@@ -30,5 +33,12 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from flask import Blueprint
+    from .routes.task_routes import tasks_bp
+    from .routes.goal_routes import goals_bp
+
+    app.register_blueprint(tasks_bp)
+    app.register_blueprint(goals_bp)
```

Comment:
The `from flask import Blueprint` will usually live at the top with the rest of the imports. Otherwise, beautiful route imports in your app init! 🎉